### PR TITLE
doc: update readme

### PR DESCRIPTION
**Pull request by setunapo**
### Description
some of the readme is a little bit outdated.

### Rationale
NA

### Example
NA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,27 +41,27 @@ To combine DPoS and PoA for consensus, Binance Smart Chain implement a novel con
  
 ### Light Client of Binance Chain
```
Comment (forcodedancing) [inline README.md:42]: Can we also rename `Binance Chain` to BNB Beacon Chain?

File: README.md [hunk 2]
```diff
@@ -41,27 +41,27 @@ To combine DPoS and PoA for consensus, Binance Smart Chain implement a novel con
  
 ### Light Client of Binance Chain  
 
-To achieve the cross-chain communication from Binance Chain to Binance Smart Chain, need introduce a on-chain light client verification algorithm.
+To achieve the cross-chain communication from Binance Chain to BNB Smart Chain, need introduce a on-chain light client verification algorithm.
```
Comment (forcodedancing) [inline README.md:44]: Also here

File: README.md (hunk 1 above)
Comment (brilliant-lx) [reply README.md]: accepted

File: README.md (hunk 2 above)
Comment (brilliant-lx) [reply README.md]: accepted

**Review by forcodedancing — APPROVED** [review]

**Review by guagualvcha — APPROVED** [review]

**Review by kyrie-yl — APPROVED** [review]

**Pull request merged**